src/integrations/prefect-kubernetes/integration_tests/src/prefect_kubernetes_integration_tests/test_eviction.py

```diff
@@ -53,18 +53,22 @@ async def test_default_pod_eviction(
         async with get_client() as client:
             updated_flow_run = await client.read_flow_run(flow_run.id)
 
-            assert updated_flow_run.state is not None, (
-                "Flow run state should not be None"
-            )
-            assert updated_flow_run.state.type != StateType.CRASHED, (
-                "Expected flow run not be marked as CRASHED, but it was"
-            )
-            assert updated_flow_run.state.type == StateType.SCHEDULED, (
-                "Expected flow run to be SCHEDULED. Got "
-                f"{updated_flow_run.state.type} instead."
-            )
-
             display.print_flow_run_result(updated_flow_run)
+            try:
+                assert updated_flow_run.state is not None, (
+                    "Flow run state should not be None"
+                )
+                assert updated_flow_run.state.type != StateType.CRASHED, (
+                    "Expected flow run not be marked as CRASHED, but it was"
+                )
+                assert updated_flow_run.state.type == StateType.SCHEDULED, (
+                    "Expected flow run to be SCHEDULED. Got "
+                    f"{updated_flow_run.state.type} instead."
+                )
```

Comment:
hm, why do we need both of these?

Reply:
This was probably left over from other testing. I'll remove the crashed check.